examples/SkeletalCartesianLambdaIntroduction.g

```diff
@@ -29,12 +29,22 @@ elf := CartesianLambdaElimination( S, R, lf );
 #! |3| → |2|
 elf = f;
 #! true
-L := MorphismsOfExternalHom( S, R );
-#! [ |3| → |2|, |3| → |2|, |3| → |2|, |3| → |2|,
-#!   |3| → |2|, |3| → |2|, |3| → |2|, |3| → |2| ]
-Li := List( L, phi -> CartesianLambdaIntroduction( phi ) );
-#! [ |1| → |8|, |1| → |8|, |1| → |8|, |1| → |8|,
-#!   |1| → |8|, |1| → |8|, |1| → |8|, |1| → |8| ]
+L := MorphismsOfExternalHom( S, R );;
+Display( L );
+#! [ { 0, 1, 2 } ⱶ[ 0, 0, 0 ]→ { 0, 1 },\
+#!  { 0, 1, 2 } ⱶ[ 1, 0, 0 ]→ { 0, 1 },\
```

Comment:
I would add a space in front for a better alignment.

Reply:
The formatting including the backslash and the reduced whitespace is currently required to make this test pass in Julia, which never inserts linebreaks itself. I tried to keep it as readable as possible. If you think it should be more readable, we can of course discuss alternatives! Do you know if one can somehow ignore whitespace in tests as in GAP?